### PR TITLE
Add return screen

### LINKED ISSUE #41: Add screen for return item

### PR DESCRIPTION
**Pull request by emmanuelroussel**
Close #41

### CODE REVIEW

File: src/mocks.ts [hunk 1]
```diff
@@ -152,6 +152,12 @@ export class InventoryDataMock {
       resolve();
     });
   }
+
+  public return(): any {
+    return new Promise((resolve, reject) => {
```
Comment (AdamVig) [inline src/mocks.ts]: I think you can replace this with `return Promise.resolve()`, but I'm not sure. This change, though minor, would reduce the line count of your mocks file by a third. 😄

File: src/pages/home/home.spec.ts [hunk 2]
```diff
@@ -47,6 +47,18 @@ describe('Home Page', () => {
     });
   }));
 
+  it('pushes rental page onNext() with \'Return\'', async(() => {
+    instance.segment = Actions.return;
+    instance.tag = 'banana';
```
Comment (AdamVig) [inline src/pages/home/home.spec.ts]: Can this test string be moved to an external "fixtures" file?

File: src/mocks.ts (hunk 1 above)
Comment (emmanuelroussel) [inline src/mocks.ts]: Good point, I will fix this in #56

File: src/pages/home/home.spec.ts (hunk 2 above)
Comment (emmanuelroussel) [inline src/pages/home/home.spec.ts]: I will also fix this in #56

**Pull request merged**